client/src/Movies/Movie.js

```diff
@@ -18,7 +18,7 @@ const Movie = (props) => {
           console.error(error);
         });
 
-  },[]);
+  },[props.match.params]);
```

Comment:
It is only three characters shorter but I like that you figured out that props.match.params worked just the same
as props.match.params.id in this instance. This shows you have a deeper understanding of the concept.

If you just pass "id" into the dependency array, I wonder if that would work??
 because its assigned to props.match.params.id on line 8.   

Cool!